src/20231205_Multi_ChaosLabsV2EthereumAndPolygonLTReductions/AaveV2Ethereum_ChaosLabsV2EthereumAndPolygonLTReductions_20231205.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV2PayloadEthereum} from 'aave-helpers/v2-config-engine/AaveV2PayloadEthereum.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+
+/**
+ * @title Chaos Labs V2 Ethereum and Polygon LT Reductions
+ * @author Chaos Labs
+ * - Snapshot: “Direct-to-AIP”
+ * - Discussion: https://governance.aave.com/t/arfc-v2-ethereum-and-polygon-lt-reductions-12-04-2023/15747
+ */
+contract AaveV2Ethereum_ChaosLabsV2EthereumAndPolygonLTReductions_20231205 is
+  AaveV2PayloadEthereum
```

Comment:
As we are not using the config engine, I would not inherit it as it would just increase the contract size. 
I would just wrap everything in an `execute()` function.